CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## 1.0.0 (2016-03-01)
```

Comment:
I would suggest writing `Mar 1, 2016` instead as `2016-03-01` could be mistaken for January 3rd.